src/vhost_user/master.rs

```diff
@@ -51,6 +51,12 @@ pub trait VhostUserMaster: VhostBackend {
     /// Setup slave communication channel.
     fn set_slave_request_fd(&mut self, fd: RawFd) -> Result<()>;
 
+    /// Retrieve shared buffer for inflight I/O tracking.
```

Comment:
Could you please help to add doc to state the ownership of `fd` will be transferred to the caller. Or should return an File object instead?

Reply:
I'm gonna follow your suggestion of putting this into a `File`, this way the ownership is obvious.